app/controllers/component-tree.js

```diff
@@ -98,7 +98,6 @@ export default Controller.extend({
    */
   pinnedObjectId: null,
   inspectingViews: false,
-  viewTreeLoaded: false,
```

Comment:
Now unneeded, since the model hook of the route now blocks until the tree is loaded.